system/jhtree/jhtree.cpp

```diff
@@ -59,7 +59,7 @@
 #include "keybuild.hpp"
 #include "layouttrans.hpp"
 
-static CKeyStore *keyStore = NULL;
+static std::atomic<CKeyStore *> keyStore(nullptr);
```

Comment:
Should this stay as it is (since the x86 code is currently correct), and wait for a consistent implementation?